Multicore hello_world sample location change (#10715)
nrfconnect/sdk-nrf: samples/multicore/hello_world/README.rst

.. _multicore_hello_world:

Multicore Hello World application
#################################

.. contents::
   :local:
   :depth: 2

> You can use this sample to see how to build Hello World application for multicore SoC.
> The sample adds remote image as a child image from the sample source files.

Reviewer: ```suggestion
The sample demonstrates how to build the Hello World application for the multicore SoC.
The sample adds a remote image as a child image from the sample source files.
```
